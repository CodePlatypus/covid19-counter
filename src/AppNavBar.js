import React from 'react';
import AppBar from "@material-ui/core/AppBar";
import IconButton from "@material-ui/core/IconButton";
import { makeStyles } from '@material-ui/core/styles';
import GitHubIcon from '@material-ui/icons/GitHub';
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";

export default function AppNavBar(props) {
    const classes = useStyles();
    return (
        <div className={classes.root}>
            <AppBar position='static' color='secondary'>
                <Toolbar variant='dense'>
                    <Typography variant='h6' color='inherit' className={classes.title}>
                        COVID-19 Counter
                    </Typography>
                    <IconButton href='https://github.com/pandabyte/covid19-counter' target='_blank' rel="noopener noreferrer">
                        <GitHubIcon />
                    </IconButton>
                    
                </Toolbar>
            </AppBar>
        </div>
    )
}

const useStyles = makeStyles(theme => ({
    root: {
      flexGrow: 1,
    },
    title: {
      flexGrow: 1,
    },
    githubLink: {
        color: 'black',
        textDecoration: 'none'
    }
}));