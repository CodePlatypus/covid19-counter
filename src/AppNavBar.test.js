import React from 'react';
import { shallow } from 'enzyme';
import AppNavBar from './AppNavBar';
import toJson from 'enzyme-to-json';

test('renders AppNavBar', () => {
    const wrapper = shallow(<AppNavBar />);
    expect(toJson(wrapper)).toMatchSnapshot();
});