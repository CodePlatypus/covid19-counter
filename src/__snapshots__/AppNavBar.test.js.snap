// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AppNavBar 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    color="secondary"
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))
      variant="dense"
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-title-2"
        color="inherit"
        variant="h6"
      >
        COVID-19 Counter
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(IconButton))
        href="https://github.com/pandabyte/covid19-counter"
        rel="noopener noreferrer"
        target="_blank"
      >
        <GitHubIcon />
      </WithStyles(ForwardRef(IconButton))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
